.github/workflows/build.yml

```diff
@@ -88,3 +88,23 @@ jobs:
           publish_dir: ./_build/doc/libportal-1/
           cname: libportal.org
           destination_dir: ./
+
+  fedora-35-abi:
+    name: Fedora 35 ABI check
```

Comment:
Will the ABI checker run differently depending on the distro version? If not, I'd recommend calling it `ABI check` only